pkg/topics/topics.go

```diff
@@ -10,15 +10,15 @@ const V1_PREFIX = "/xmtp/0/"
 const V3_PREFIX = "/xmtp/1/"
```

Comment:
This is what needs to be updated
```suggestion
const V3_PREFIX = "/xmtp/mls/1/"
```

Reply:
Ah, good point!